test/commons/standards/get-aria-roles-by-type.js

```diff
@@ -1,59 +1,32 @@
 describe('standards.getAriaRolesByType', function() {
   var getAriaRolesByType = axe.commons.standards.getAriaRolesByType;
 
-  before(function() {
-    axe._load({});
-  });
+  it('should return a list of role names by type', function() {
```

Comment:
I got tired of updating this test when adding new roles. This should allow us to not have it change so often.